students/itsabhi/Lesson8/circle.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
+
+import math
+
+
+class Circle:
+    # Circle class
+    _diameter = None  # This cannot be stored as its not an object attribute
+    _area = None  # This cannot be stored as its not an object attribute
```

Comment:
what is the purpose of these?

Reply:
I might have copied these from the class lesson example:
class C:
    _x = None
    @property
    def x(self):
        return self._x